Do not update the updater separately

frauzufall opened the pull request:
The removed code does the following:
  * check if the updater can be updated
  * if yes, download updater and it's dependencies and install it, do not ask for approval
  * try to restart the updater in place
  * if that does not work, ask for restart of Fiji

This is why it should be removed:
  * if other components depend on the updater, this can break the updater because the other component will not get updated at the same time as the updater
  * the updater will be updated without the users approval

ctrueden commented:
The reason for the failing Travis build is because Travis CI recently changed the default environment from trusty to xenial. It seems the xvfb config doesn't work for xenial as written. As a quick fix, we can add `dist: trusty` to the top of `.travis.yml`. But it would be good to investigate why it doesn't work on xenial, and ideally update it to work, so that we aren't blindsided if/when trusty stops being supported.

frauzufall commented:
Thank you for pointing me to the right direction @ctrueden! I removed the lines in the travis config which are not needed any more in xenial and reverted the commit order so that there is no failing commit. Travis is happy now.

ctrueden commented:
I asked Johannes why this logic was introduced. Here is what he said:

> The reason was dependency hell. If you don't update the updater first, then create a new class loader, then re-load the updater and run it, it really can break _everything_, especially when imagej-ui-swing is updated (and not necessarily the updater).
